### DISCUSSION TITLE
Overriding Data from Entries Fieldtype in CP

### DISCUSSION THREAD

**Discussion by xiano8494** [statamic/cms#4552] [Q&A]
I want to be able to optionally override some of the data being passed from a linked set of entries on a page. Essentially I want to have a default description, but be able to edit that description on some of the pages if I need to.
I have added an "Entries" field to the blueprint for the page which allows me to link to the items in the collection, but I can't see a sensible way to achieve what I need.
I did wonder if I could create a "replicator", containing an entry field (only allowing a single item) and then adding a field that could be used to override the description, which I think would work, but seems a little messy.  If this is the best way, it would be really nice if I could populate the "override" field with the content from the associated entry...

Thoughts? Is there a better way?

**Comment by jasonvarga** [comment]
What you're suggesting would work. If you need a less cluttered UI, you could use `grid` instead.

Create a grid with two fields: the `entries` field with `max_items: 1`, then a text field for what you want to override.

**Reply by xiano8494** [reply]
Ah, yes, that could be neater.
Is there a way to make the text field populate based on the entry chosen?

**Reply by jasonvarga** [reply]
Not without creating some sort of custom fieldtype.
